src/main/java/de/uni_tuebingen/ub/nppm/db/EinzelbelegDB.java

```diff
@@ -24,7 +28,7 @@ public static List getListTextKritik() throws Exception {
     }
 
     public static Einzelbeleg getFirstPublicEinzelbeleg() throws Exception {
-        try (Session session = getSession()) {
+        try ( Session session = getSession()) {
```

Comment:
Please remove whitespace between `(` and `Session` (multiple occurances)

Reply:
done